sass/_component_docutils.scss

```diff
@@ -62,3 +53,33 @@ table.docutils {
     }
     margin-bottom: $spacer;
 }
+
+// Figures
+.figure {
```

Comment:
Perhaps we should consider extracting these styles into their own `.scss` files?

That would provide more long term structure I think.